.npmignore

```diff
@@ -0,0 +1,8 @@
+*.log
+*.tmp
+coverage/
+h3c/
+node_modules/
+.*
+build/
+test/
```

Comment:
The main issue in this file is *not* including `dist` - without this file npm defaults to using `.gitignore`, which ignores `dist` and therefore breaks the distribution.

Reply:
Maybe add a comment to these two files to preserve this knowledge? I'm not sure if using npmignore would be expected.

Reply:
Whoops, missed this comment. I think this is actually pretty standard for module publishers, I just missed it in the first go-round.